cassandra/cluster.py

```diff
@@ -3521,11 +3523,17 @@ def _reconnect_internal(self):
             try:
                 return self._try_connect(host)
             except ConnectionException as exc:
-                errors[str(host.endpoint)] = exc
+                if os.environ.get('TRACEBACK','FALSE').lower() == "true":
+                    errors[str(host.endpoint)] = traceback.format_exc()
```

Comment:
Does this `traceback.format_exc()` include the error `exc`?
If no - we can do concatenation:
```
                    errors[str(host.endpoint)] = exc + traceback.format_exc()
```

Reply:
Yes, `traceback.format_exc()` includes the exception information and stack trace both.